src/planscape/planning/views_v2.py

```diff
@@ -140,3 +142,11 @@ def toggle_status(self, request, planningarea_pk, pk=None):
         toggle_scenario_status(scenario, self.request.user)
         serializer = ScenarioSerializer(instance=scenario)
         return Response(data=serializer.data)
+
+
+class CreatorViewSet(ReadOnlyModelViewSet):
```

Comment:
I'm not sure if this needs to be a separate ViewSet, but it felt like it deserved its own.